harnessed_jobs/fe55_raft_acq_sim/v0/producer_fe55_raft_acq_sim.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+""" Producer script """
+
+from simulation import fake_raft
+
+# These are specific to this test
+TESTTYPE = 'FE55'
+IMGTYPES = ['BIAS', 'FE55']
+PROCESS_NAME_IN = 'vendorIngest'
+PATTERN = '*_fe55*.fits'
+
+RAFT = fake_raft.Raft.create_from_etrav(fake_raft.RAFT_ID)
+
+for image_type in IMGTYPES:
+    RAFT.file_copy(PROCESS_NAME_IN, fake_raft.OUTPATH,
```

Comment:
Here I'd prefer simply `'.'` to `fake_raft.OUTPATH`.   Otherwise, I'd have to look up what `.OUTPATH` is to understand this code.